### PR TITLE
Use the npm version of pontoon-to-webext.

### PR DESCRIPTION
**Pull request by bwinton**
Note: This is untested, because my copy of MINGW64 doesn't have make.  😞
Note 2: I've upstreamed the changes from your repo into the npm version.  🙂

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,14 +48,11 @@
...
     "geckodriver": "1.4.0",
-    "habitat": "3.1.2",
-    "minimist": "1.2.0",
     "mocha": "3.2.0",
     "node-sass": "4.5.0",
     "npm-run-all": "4.0.2",
     "nsp": "2.6.3",
-    "properties-parser": "0.3.1",
-    "q-io": "1.13.2",
+    "pontoon-to-webext": "^1.0.2",
```
Comment (6a68) [inline package.json]: Could this be an exact version number?

**Comment by 6a68** [comment]
thanks, @bwinton! looking now

**Comment by 6a68** [comment]
sweet, the `make addon_locales` command works for me.

@bwinton happy to merge this once you make the version number exact in package.json

File: package.json (hunk 1 above)
Comment (ianb) [inline package.json]: I'm sorting through PRs, I'll just land and update this part

Comment (6a68) [inline package.json]: :+1:

**Pull request merged**